### PR TITLE
make PC police happy

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@blowdart  can you verify my changes are OK?

File: aspnet/security/authorization/policies.rst [hunk 1]
```diff
@@ -95,7 +95,7 @@ Our minimum age handler might look like so;
      }
  }
 
-In the code above we first look to see if the current user principal has a date of birth claim which has been issued by an Issuer we know and trust. If the claim is missing we can't authorize so we return. If we have a claim we figure out how old the user is, and if they meet the minimum age passed in by the requirement then authorization has been successful, so we call ``context.Succeed()`` passing in the requirement that has been successful as a parameter.
+In the code above we first look to see if the current user principal has a date of birth claim which has been issued by an Issuer we know and trust. If the claim is missing we can't authorize so we return. If we have a claim, we figure out how old the user is, and if they meet the minimum age passed in by the requirement then authorization has been successful. Once a claim is successful we call ``context.Succeed()`` passing in the requirement that has been successful as a parameter.
```
Comment (danroth27) [inline aspnet/security/authorization/policies.rst]: > Once a claim is successful

Once authorization is successful

**Comment by danroth27** [comment]
Minor comment, but otherwise :shipit:

**Comment by blowdart** [comment]
:ship:

**Comment by Rick-Anderson** [comment]
merged

**Pull request closed without merging**